xdsl/dialects/omp.py

```diff
@@ -229,6 +243,61 @@ def verify(self, op: Operation) -> None:
         return super().verify(op)
 
 
+class BlockArgOpenMPOp(IRDLOperation):
```

Comment:
We use the `Operation` suffix as a naming convention for abstract base classes.
```suggestion
class BlockArgOpenMPOperation(IRDLOperation, ABC):
```